File: android/versioned-abis/expoview-abi34_0_0/src/main/java/abi34_0_0/expo/modules/analytics/amplitude/AmplitudePackage.java
package abi34_0_0.expo.modules.analytics.amplitude;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi34_0_0.org.unimodules.core.BasePackage;
import abi34_0_0.org.unimodules.core.ExportedModule;

public class AmplitudePackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new AmplitudeModule(context));
  }
}

File: android/versioned-abis/expoview-abi43_0_0/src/main/java/abi43_0_0/expo/modules/analytics/amplitude/AmplitudePackage.kt
package abi43_0_0.expo.modules.analytics.amplitude

import android.content.Context
import abi43_0_0.expo.modules.core.BasePackage
import abi43_0_0.expo.modules.core.ExportedModule

class AmplitudePackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> {
    return listOf(AmplitudeModule(context))
  }
}
